dateparser/custom_lang_detect/__init__.py

```diff
@@ -0,0 +1,19 @@
+import importlib
+
+loaded_parser = None
+language_parser_module = None
+
+def language_parser(paser_module="fast_text"):
+  global loaded_parser
+  global language_parser_module
+
+  try:
+    if loaded_parser == None:
+      paser_module = importlib.import_module("dateparser.custom_lang_detect." + paser_module)
+      language_parser_module = paser_module.language_parser
+      print("custom loader init")
+  except:
```

Comment:
same here, we should specify which exception are we capturing instead of capturing all :)